Dockerfile

```diff
@@ -0,0 +1,32 @@
+# WARNING: THIS FILE IS MANAGED IN THE 'BOILERPLATE' REPO AND COPIED TO OTHER REPOSITORIES.
```

Comment:
does this need a dockerfile?  if so, should we add this https://github.com/lyft/flyteadmin/blob/master/Makefile#L2 to the makefile which would add this https://github.com/lyft/boilerplate/blob/master/boilerplate/lyft/docker_build/Makefile

Reply:
no dockerfile needed, removing